src/components/atoms/Button/CustomButton.tsx

```diff
@@ -0,0 +1,17 @@
+import {Pressable, Text, View} from 'react-native';
+import {styles} from './style';
+
+interface ICustomButtonProps {
+  title: string;
+  onPress: () => void;
+}
+
+export const CustomButton = ({title, onPress}: ICustomButtonProps) => {
+  return (
+    <Pressable onPress={onPress}>
+      <View style={[styles.button, styles.elevation]}>
```

Comment:
Potrzebne ci to view? Nie możesz pressable ostylować?

Reply:
done